app/src/main/java/com/kickstarter/ui/activities/CommentsActivity.kt

```diff
@@ -27,10 +46,48 @@ class CommentsActivity :
         setContentView(view)
         binding.commentsRecyclerView.adapter = adapter
 
+        val loadMoreListView = PaginationHandler(
```

Comment:
From line 46 to line 61 could it be extracted to a function 
`fun setUpPagination` or any other name.

Reply:
Done